app/va/va_profile/va_profile_client.py

```diff
@@ -12,10 +14,22 @@
 from time import monotonic
 from typing import Dict, List, TYPE_CHECKING
 
+
 if TYPE_CHECKING:
     from app.models import RecipientIdentifier
     from va_profile_types import ContactInformation, CommunicationPermissions, Profile, Telephone
 
+EMAIL_TYPE = 'email'
+LETTER_TYPE = 'letter'
+MOBILE_TYPE = 'mobile'
+PUSH_TYPE = 'push'
+SMS_TYPE = 'sms'
+
+VA_NOTIFY_TO_VA_PROFILE_NOTIFICATION_TYPES = {
+    EMAIL_TYPE: 'Email',
+    SMS_TYPE: 'Text',
+}
```

Comment:
Ditto.  These is immediately after the above info in the link I shared.

Reply:
@k-macmillan What is the reason for not importing?

Reply:
These should never have been in models.py and will be removed when we tear out the old code @kalbfled . They are only used by the VA Profile class and using them requires an import inside the function to avoid circular references.